app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/model/MainDialogModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.mobile.tandil.kotlinbaseproject.mvvm.model
+
+class MainDialogModel {
+    private var parkingSpaces = ""
```

Comment:
do not hardcode, please us a constant for this

Reply:
done